CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ straightforward as possible.
 ### Fixed
 - mobile upload on Android
 - device log. new format support [#565]
+- Log in by key uploading UI issue [#583]
```

Comment:
in a fresh release please ....  ie after line 18

Reply:
done